### PR TITLE
Added AI course in english

### PR DESCRIPTION
**Pull request by bhumiiagarwal**
What does this PR do?
Add resource(s)

For resources
Description
Added a course in english on AI

Why is this valuable (or not)?
Yes, it is valuable.

How do we know it's really free?
It is available on coursera.

For book lists, is it a book? For course lists, is it a course? etc.
It is a course

Checklist:
 Read our [contributing guidelines](https://github.com/EbookFoundation/free-programming-books/blob/main/docs/CONTRIBUTING.md).
 [Search](https://ebookfoundation.github.io/free-programming-books-search/) for duplicates.
 Include author(s) and platform where appropriate.
 Put lists in alphabetical order, correct spacing.
 Add needed indications (PDF, access notes, under construction).
 Used an informative name for this pull request.
Follow-up
Check the status of GitHub Actions and resolve any reported warnings!

### CODE REVIEW

**Comment by bhumiiagarwal** [comment]
@eshellman @sohansai please review this PR

File: courses/free-courses-en.md [hunk 1]
```diff
@@ -264,6 +264,7 @@
 * [MIT Deep Learning and Artificial Intelligence Lectures](https://deeplearning.mit.edu) - Lex Fridman, et al.
 * [MIT's Artificial Intelligence](http://ocw.mit.edu/courses/electrical-engineering-and-computer-science/6-034-artificial-intelligence-fall-2010/) - Prof. Patrick Henry Winston (MIT OpenCourseWare)
 * [Stanford CS221: Artificial Intelligence: Principles and Techniques \| Autumn 2019](https://www.youtube.com/playlist?list=PLoROMvodv4rO1NB9TD4iUZ3qghGEGtqNX) - Stanford Online
+* [Artificial Intelligence Full Course 2022](https://www.youtube.com/watch?v=lTzHlU3OrXs) - Simplilearn
```
Comment (sohansai) [inline courses/free-courses-en.md:267]: We prefer 'playlist' links for YouTube contributions, and be sure to include the full title as provided by the resource

**Comment by bhumiiagarwal** [comment]
@sohansai, i have updated this with a coursera course

**Comment by eshellman** [comment]
limited duration course

**Pull request closed without merging**